src/sentry/api/endpoints/organization_events.py

```diff
@@ -432,12 +441,7 @@ def _dashboards_data_fn(scopedDataset, offset, limit, scoped_query, dashboard_wi
                 if has_errors and has_other_data and not using_metrics:
                     # In the case that the original request was not using the metrics dataset, we cannot be certain that other data is solely transactions.
                     sentry_sdk.set_tag("third_split_query", True)
-                    transactions_only_query = (
-                        f"({scoped_query}) AND event.type:transaction"
-                        if scoped_query
-                        else "event.type:transaction"
-                    )
```

Comment:
why was this removed?

Reply:
For this block of code we're evaluating the `transactions` only path without metrics, so I removed the call to the discover dataset and instead queried transactions directly. Is there something I'm missing the requires us to hit discover still in this case?

Reply:
Can we gate the using the transaction fallback behind the `split-dataset` flag?

Reply:
Oh actually, this should be okay since we're explicitly trying to query transactions here. Can you send [this](https://github.com/getsentry/sentry/pull/73893/files#diff-084d1fb1e68307632ee00ae568af3c10195b19880bc8b28c49585c1e1f4d5beeR407-R416) query to the errors dataset too?